### ISSUE TITLE
Error with compiling grammar

### ISSUE THREAD

**Issue by chuckberrypi** [gleam-lang/gleam-mode#26]
I followed the instructions for installing `gleam-ts-mode` by cloning the repository to `~/.config/emacs/modules/lang/`, adding the `(use-package 'gleam-ts-mode ...)` snippet to my `~/doom.d/packages.el`, and reloading that file. When I went to run `M-x: gleam-ts-install-grammar`, I got the following error: 

`⛔ Warning (treesit): The installed language grammar for gleam cannot be located or has problems (not-found): (libtree-sitter-gleam.so libtree-sitter-gleam.so.0 libtree-sitter-gleam.so.0.0 libtree-sitter-gleam.dylib libtree-sitter-gleam.dylib.0 libtree-sitter-gleam.dylib.0.0) No such file or directory`

Any suggestions?

**Comment by J3RN** [comment]
This error comes from [inside Emacs' `treesit` library](https://github.com/emacs-mirror/emacs/blob/a154f0aa73bceeaaeefcd243e7db9d09e3f450d7/lisp/treesit.el#L3621).  It seems to indicate that the grammar installation failed.  If you look in the `*Messages*` buffer after running this command, is there anything relevant in there?

**Comment by chuckberrypi** [comment]
Yeah, it says `Library installed to ~/.config/emacs/.local/cache/tree-sitter/libtree-sitter-gleam.dylib`, and when I go to that directory the `libtree-sitter-gleam.dylib` file is indeed there. I double checked, and `user-emacs-directory` does evalutate to `~/.config/emacs/.local/cache`, so I'm really not sure why tree-sitter can't see it.

**Comment by chuckberrypi** [comment]
closing since it sounds like this isn't an issue with this repo
